analytical_engine/core/fragment/arrow_projected_fragment.h

```diff
@@ -916,6 +916,23 @@ class ArrowProjectedFragment
   }
 
   inline bool directed() const { return directed_; }
+#ifdef ENABLE_JAVA_SDK
```

Comment:
These interfaces can improve java graph traverse speed tremendously be returning a pointer rather than stakc-allocated object. @sighingnow @yecol Is it okay for us to do this? Thanks.

Reply:
I suggest to leave them here without `ENABLE_JAVA_SDK`. We already have some other "unsafe" & "raw" interfaces in fragment.